app/src/test/java/decide/DecideTest.java

```diff
@@ -45,4 +45,122 @@
         boolean result = Decide.decide(points, params, lcm, puv);
         assertTrue("Simple test case with LIC0 and LIC1", result);
     }
+
+
+
+
+
+
+
+     //False test cases that combines LIC 0 4 8
+
+     @Test public void falseTestWherePointsForLIC0AreNotConsecutive() { 
+
+        Point points[] = new Point[5];
+        // points[0] and points[4] are apart more than LENGHT1=10 
+        // but because they are not consecutive the algorithm returns false.
+        points[0] = new Point(2, 2);
+        points[1] = new Point(4, 4);
+        points[2] = new Point(6, 6);
+        points[3] = new Point(8, 8);
+        points[4] = new Point(-10, -10); 
```

Comment:
LIC 0 will be true since the 2 last points are a distance of 18 * sqrt(2) (> 10) apart.
I ran the test and the "false" output results from a false LIC8. Because of a rounded value, the area of the triangle comes out megative. To solve the problem, you can change the function getCircumscribedCirclesRadius in the Functions class and call the function getAreaTriangle instead of the calculation in the function.